core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -194,7 +194,8 @@ private TableProperties() {
 
   public static final String METRICS_MODE_COLUMN_CONF_PREFIX = "write.metadata.metrics.column.";
   public static final String DEFAULT_WRITE_METRICS_MODE = "write.metadata.metrics.default";
-  public static final String DEFAULT_WRITE_METRICS_MODE_DEFAULT = "truncate(16)";
```

Comment:
This is changing a public API and to a new type. May need to reconsider this. Would probably be safer to just add the new variable below with a slightly different name?

Although if other folks don't mind, I think that's fine.

Reply:
Sure, changed it back, it made the diff smaller